layers/+lang/python/README.org

```diff
@@ -306,6 +309,14 @@ To use =dap-mode= for debugging do:
   pip install "ptvsd>=4.2"
 #+END_SRC
 
+** Notebook and code cells
+To use =code-cells-mode= with ipython notebook automatically you should install
+[[https://github.com/mwouts/jupytext][jupytext]]. Make sure that the
+=ipython-notebook= layer is not used.
+
+Then you can evaluate the current cell in a REPL process with ~SPC s c~ or all
+the cells above with ~SPC s a~.
```

Comment:
These bindings don't look right, they should go under `SPC m s c` and `SPC m s a` at least that's what your code will bind them at.

Reply:
Yes, that was my mistake.